### PR TITLE
Refactor __init of models

### PR DESCRIPTION
**Pull request by jamadaha**

### CODE REVIEW

File: JANGAN/Classifier/ClassifierMLModel.py [hunk 1]
```diff
@@ -48,25 +48,40 @@ def SetupModel(self):
...
-            self.Classifier.compile(
-                optimizer=keras.optimizers.Adam(learning_rate=classSchedule),
-                loss_fn=keras.losses.CategoricalCrossentropy(from_logits=True),
-            )  
-
-        self.Trainer = ct.ClassifierTrainer(self.Classifier, self.TensorDatasets, self.EpochCount, self.RefreshEachStep, self.SaveCheckpoints, self.CheckpointPath, self.LatestCheckpointPath, self.LogPath)
-
+    
+    def __GetLossFunction(self): 
+        return keras.losses.BinaryCrossentropy(from_logits=True)
```
Comment (kris701) [inline JANGAN/Classifier/ClassifierMLModel.py:83]: Incorrect loss function! Use `CatagorialCrossentropy`

**Review by kris701 — CHANGES_REQUESTED** [review]

**Review by kris701 — APPROVED** [review]
